ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "v1.0",
+    "author": "ModelArts Service of Huawei Cloud",
+    "authorEmail": "william.zhouweimin@huawei.com",
+    "framework": "TensorFlow 1.8.0",
+    "codeURL": "https://github.com/huawei-clouds/modelarts-example/DAWNBench_Inference",

```

Comment:
This URL gives a 404 Error

Reply:
Hi, thanks for your efforts on this PR.
The right code URL and commit hash have been updated.